app/state.go

```diff
@@ -30,6 +30,8 @@ import (
 )
 
 const (
+	errMsgDependencyNotSatisfiedF = "Artifact dependency %q not satisfied " +
```

Comment:
Question: What is with the `F` ending?

Reply:
Format string, just a personal programming-convention.